requirements.txt

```diff
@@ -1,8 +1,9 @@
-numpy==1.13.1
+numpy==1.13.3
 matplotlib==2.0.2
 pyqtgraph==0.10.0
 PyVISA==1.8
-PyQt5==5.9
```

Comment:
Travis can't install this. Does that not mean that our CI will stop working?

Reply:
I know its because that version is only out for win and OSx not linux. I have reverted this